app/views/staff/activities/show.html.haml

```diff
@@ -27,3 +27,14 @@
   - if @activity.project? && policy(:project).create?
     = render partial: "staff/shared/activities/transactions", locals: { activity: @activity, transaction_presenters: @transaction_presenters }
     = render partial: "staff/shared/activities/budgets", locals: { activity: @activity, budget_presenters: @budget_presenters }
+
+  - if @activity.project?
+    .govuk-grid-row
```

Comment:
After the recent changes to this page I reckon we can partial this up in.

If only only users who belong to delivery partners should be able to create and edit these then a test should help drive this into the above block that checks `@activity.project? && policy(:project).create?` and within the partial a further check for `:edit?`

We might be able to also show the list of implementers to everyone so it'd be separate?